pyproject.toml

```diff
@@ -46,9 +49,6 @@ test = [
   "aiodataloader",
   "graphql-sync-dataloaders;python_version>\"3.7\"",
 ]
-asgi-file-uploads = ["python-multipart>=0.0.5"]
-tracing = ["opentracing"]
-telemetry = ["opentelemetry-api"]
```

Comment:
why was this removed? I can see a code for these packages in the repo

Reply:
@drozdowsky 
The optional requirements have been restored and new version was [released](https://github.com/mirumee/ariadne/releases/tag/0.26.1).

Thanks for noticing this